scripts/publish-documentation/README.md

```diff
@@ -0,0 +1,152 @@
+### Description
+
+This directory contains a script and serves as a workspace for it. The script generates and publishes 
+Dokka documentation for 'old'(`<2.0.0`) releases of Spine projects. Documentation is published to the 
```

Comment:
I would say "previous" or "already published" instead of "old". They are hardly old. 1.8.x is the most recent one.